### PR TITLE
calculateRentalCost

### PR DESCRIPTION
**Pull request by svyatoslavPy**
# Transportation on Vacation

After a hard quarter in the office you decide to get some rest on a vacation. So you
will book a flight for you and your family and try to leave all the mess behind you.

You will need a rental car in order for you to get around in your vacation. The
manager of the car rental makes you some good offers.

Every day you rent the car costs `$40`. If you rent the car for `7` or more days, you
get `$50` off your total. Alternatively, if you rent the car for `3` or more days,you
get `$20` off your total.

Implement `calculateRentalCost` function that returns the total amount for different
number of days.

Examples:
```js
calculateRentalCost(1); // 40
calculateRentalCost(3); // 100
calculateRentalCost(7); // 230
```

## Instructions
**Read [the guideline](https://mate-academy.github.io/js_task-guideline/) before start**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
+  const DISCOUNT_THREE_DAYS = 20;
+  const BASE_PRICE_OF_RENT = 40;
+
+  const baseResult = days * BASE_PRICE_OF_RENT;
+
+  if (days >= 3 && days <= 6) {
+    return baseResult - DISCOUNT_THREE_DAYS;
+  }
+
+  if (days >= 7) {
```
Comment (lerastarynets) [inline src/calculateRentalCost.js:21]: pls move ALL numbers to consts and use here

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Nice start! Let's fix the solution according to the checklist

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -1,10 +1,32 @@
...
+
 function calculateRentalCost(days) {
   // write code here
+  const DISCOUNT_SEVEN_DAYS = 50;
+  const DISCOUNT_THREE_DAYS = 20;
+  const BASE_PRICE_OF_RENT = 40;
+
+  const LONG_TERM = 7;
+  const LOW_TERM = 3;
+  const MIDDLE_TERM = 6;
```
Comment (anastasiiavorobiova) [inline src/calculateRentalCost.js:17]:
```suggestion

```

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -1,10 +1,32 @@
...
+
+  const baseResult = days * BASE_PRICE_OF_RENT;
+
+  if (days >= LOW_TERM && days <= MIDDLE_TERM) {
```
Comment (anastasiiavorobiova) [inline src/calculateRentalCost.js:21]:
```suggestion
  if (days >= LOW_TERM && days < LONG_TERM) {
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
GJ! Almost done!

**Review by lerastarynets — APPROVED** [review]
You've got this, keep pushing forward!

**Pull request closed without merging**